README.md

```diff
@@ -3,10 +3,20 @@
 ## How to approach this repo
 The intention behind this project is discovery and understanding. The idea is to explore how the internet is put together and use common industry tools to see and understand all the communications necessary to make the services and apps we use work.
 
-We decided to take notes that creates a reproducable experience. We document our questions, discovery, and our processes so other people who may want to take a similar journey can have a path to follow. The `notes` folder is the home of this documentation. There is, at least as of yet, very little actual software to run from this repo. Instead, the intent is to guide the experience and the learning process.
+We decided to take notes that creates a reproducable experience. We document our questions, discovery, and our processes so other people who may want to take a similar journey can have a path to follow. We are breaking up our exploration into chapters. Each chapter will have a README that documents our exploration and the final state for that chapter of whatever files we create in our exploration. There is, at least as of yet, very little actual software to run from this repo. Instead, the intent is to guide the experience and the learning process.
```

Comment:
"...are breaking" or "...have broken" ? "...will have a README" or "...has a README" ? I assume that by the time someone actually reads this thing who is not us, these decisions will have been done and the work will be complete?